backend/routes/seatbid.js

```diff
@@ -121,8 +135,25 @@ router.post('/accept', authenticateUser, async (req, res) => {
             }),
         ]);
 
-        activeBiddings.delete(biddingId);
-        bids.delete(biddingId);
+        seatsToRemove.forEach(seatNumber => {
```

Comment:
@nowshin-108 Just to confirm. This is to delete other bids associated to this seat if it was accepted in a bidding, right?

Reply:
Yeap!